### PR TITLE
DDT-868: CargoItem and ShipmentEquipment is required on SI

### PR DESCRIPTION
**Pull request by nt-as**
Signed-off-by: Niels Thykier <nt@asseco.dk>

### CODE REVIEW

**Comment by nt-as** [comment]
Relevant test changes are in https://github.com/dcsaorg/DCSA-EBL/pull/153

File: src/main/java/org/dcsa/core/events/model/transferobjects/ShipmentEquipmentTO.java [hunk 1]
```diff
@@ -18,12 +19,14 @@
 	@NotNull(message = "Equipment is required.")
 	EquipmentTO equipment;
 
-	@Valid
-	private List<CargoItemTO> cargoItems;
+  @Valid
+  @NotNull
+  @Size(min = 1)
```
Comment (pimg) [inline src/main/java/org/dcsa/core/events/model/transferobjects/ShipmentEquipmentTO.java:24]: NotNull and Size can be simplified with @NotEmpty

File: src/main/java/org/dcsa/core/events/model/transferobjects/ShippingInstructionTO.java [hunk 2]
```diff
@@ -26,6 +28,8 @@
 
   @JsonProperty("utilizedTransportEquipments")
   @Valid
+  @NotNull
+  @Size(min = 1)
```
Comment (pimg) [inline src/main/java/org/dcsa/core/events/model/transferobjects/ShippingInstructionTO.java:32]: same as above

**Review by pimg — COMMENTED** [review]
some small comments

File: src/main/java/org/dcsa/core/events/model/transferobjects/ShipmentEquipmentTO.java (hunk 1 above)
Comment (nt-as) [reply src/main/java/org/dcsa/core/events/model/transferobjects/ShipmentEquipmentTO.java]: Fixed

File: src/main/java/org/dcsa/core/events/model/transferobjects/ShippingInstructionTO.java (hunk 2 above)
Comment (nt-as) [reply src/main/java/org/dcsa/core/events/model/transferobjects/ShippingInstructionTO.java]: Fixed

**Review by pimg — APPROVED** [review]

**Comment by douradofrazer** [comment]
Should we add a message in the validation annotation? It helps for the returned error response.

**Pull request merged**